ansible/ci/check_eessi.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Run EESSI test job
+  hosts: login # TODO: Limit this to single node when there are multiple?
+  become: true
+  become_user: testuser
```

Comment:
TBH I think I'd be happier decoupling this from the CI-defined `testuser`. We can take the same approach as `hpctests` and just explicitly create a /home/eessi_tests directory owned by `{{ ansible_user }}`, so no become needed. Its easier to run on other environments then.